cmd/bisync/bilib/hash.go

```diff
@@ -0,0 +1,57 @@
+// Package bilib provides common stuff for bisync and bisync_test
+// Functions here provide simple aliases for rclone hash names.
+// TODO will be later implemented in fs/hash.go
+// SEE https://github.com/rclone/rclone/issues/5071
```

Comment:
@ncw 

This whole kludge will have gone after I submit a PR for #5071

Reply:
Simple hash aliases were merged in core by #5339.
So I removed this kludge by https://github.com/rclone/rclone/pull/5164/commits/fcb9cbe1dbdfb7e4ac892f1860e5e2b1d155d1b1.